docs-website-src/content/_index.md

```diff
@@ -218,6 +218,52 @@ Remove the `elevate-cpanel` service used during the upgrade process.
 
 A final reboot is performed at the end of this stage.
 
+## Advanced Otions
+
+### Controlling leapp upgrade manually
+
+By default the elevate script runs the leapp process.
+But in some special cases you might want to control the leapp process yourself
+on the hypervisor or server.
+
+Using the `--no-leapp` leaves you the control to run the leapp process on your own.
+Instead of running leapp, the elevation process is paused.
+
+Once leapp has been successfully performed on the server/hypervisor you can then remove the file
```

Comment:
leapp should not be mentioned here. Instead: 
```
Once the system has been upgraded from 7 to 8...
```

Reply:
fixed